.github/workflows/linters.yml

```
@@ -51,6 +51,47 @@ jobs:
         run: npx envinfo
       - name: Lint C/C++ files
         run: make lint-cpp
+<<<<<<< HEAD
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)